acnportal/acnsim/network/charging_network.py

```diff
@@ -1,17 +1,26 @@
-from .constraint_set import ConstraintSet
-
+from .current import Current
+import pandas as pd
+import numpy as np
 
 class ChargingNetwork:
     """
     The ChargingNetwork class describes the infrastructure of the charging network with
     information about the types of the charging station_schedule.
     """
 
-    def __init__(self):
+    def __init__(self, constraints=None, magnitudes=None):
```

Comment:
I am unsure if we should allow constraints / magnitudes to be passed as arguments. I think I prefer adding them one at a time so we can ensure that the number of constraints and number of magnitudes match... 

I'm open to it though, we have a good use case.

Reply:
We won't allow passing constraints/magnitudes in the construction of the network, there's not a good use case we can think of.